conf/reef/rados/test-confs/pool-configurations.yaml

```diff
@@ -14,7 +11,6 @@ replicated:
     pool_type: replicated
     pool_name: re_pool_30
     pg_num: 128
-    size: 3
```

Comment:
Please make changes to other versions of this file as well

Reply:
Updated the files